src/bsp/platform/posix/xi_bsp_io_fs_posix.c

```diff
@@ -52,39 +53,39 @@ static xi_state_t xi_fs_posix_errno_2_xi_state( int errno_value )
         case EFAULT:
         case ENOENT:
         case ENOTDIR:
-            ret = XI_FS_RESOURCE_NOT_AVAILABLE;
+            ret = XI_BSP_IO_FS_RESOURCE_NOT_AVAILABLE;
             break;
         case ELOOP:
         case EOVERFLOW:
-            ret = XI_FS_ERROR;
+            ret = XI_BSP_IO_FS_ERROR;
             break;
         case ENAMETOOLONG:
-            ret = XI_INVALID_PARAMETER;
+            ret = XI_BSP_IO_FS_INVALID_PARAMETER;
             break;
         case ENOMEM:
-            ret = XI_OUT_OF_MEMORY;
+            ret = XI_BSP_IO_FS_OUT_OF_MEMORY;
             break;
         default:
-            ret = XI_FS_ERROR;
+            ret = XI_BSP_IO_FS_ERROR;
     }
 
     return ret;
 }
 
 /* helper function that translates posix stat to xi stat */
-static xi_state_t xi_fs_posix_stat_2_xi_stat( const struct stat* const posix_stat,
-                                              xi_fs_stat_t* const xi_stat )
+static xi_bsp_io_fs_state_t xi_bsp_io_fs_posix_stat_2_xi_stat( const struct stat* const posix_stat,
```

Comment:
xi_fs_posix_stat_2_xi_bsp_io_fs_posix_stat?

Reply:
+ ohnos there are two types with slightly same name, easy to mix them up:
- xi_bsp_io_fs_state_t
- xi_bsp_io_fs_stat_t
No problem here, just hmm... I don't know. It can stay if you are ok with them.